### ISSUE TITLE
Transfer Weights in Ultralytics Cloud

### ISSUE THREAD

**Issue by wlangkau** [ultralytics/hub#913]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and [discussions](https://github.com/ultralytics/hub/discussions) and found no similar questions.

### Question

How can I use the Ultralytics Cloud (including the hardware resources) for transfer learning?

I want to use your pretrainied yolo11x.pt model with my own uploaded dataset (incl. images, labels and my data.yaml) but I don't want to loose the 80 weights from COCO8 that have been trained on yolo11x.
Is this possible with the new Ultralytics Cloud?

In COLAB I would do something like the following, but then I need to pay the Google hardware again. Can I apply this in the Ultralytics Cloud somehow?
-    model = YOLO("yolo11n.yaml").load("yolo11n.pt")  # build from YAML and transfer weights
-    results = model.train(data="MY_DATASET.yaml", epochs=100, imgsz=640)

And what would be the references for "MY_DATASET.yaml" pointing to my Ultralytics Cloud dataset in the COLAB code?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @wlangkau, thank you for raising an issue about [Ultralytics HUB](https://github.com/ultralytics/hub) 🚀! Your interest in leveraging Ultralytics Cloud for transfer learning is exciting! An Ultralytics engineer will be with you soon to assist further. In the meantime, please check out our [HUB Docs](https://docs.ultralytics.com/hub) for more insights:

- [**Quickstart**](https://docs.ultralytics.com/hub/quickstart): Kick off your journey with HUB and start training and deploying your models in no time.
- [**Datasets: Preparing and Uploading**](https://docs.ultralytics.com/hub/datasets): Learn the seamless process of getting your datasets ready and uploaded, fully optimized for YOLO.
- [**Models: Training and Exporting**](https://docs.ultralytics.com/hub/models): Get insights on training and exporting models, including transferring weights, to ensure your models are ready for deployment.
- [**Inference API**](https://docs.ultralytics.com/hub/inference-api): Discover how to leverage the Inference API for running cloud-based predictions with your trained models.

For your detailed questions about dataset references and maintaining COCO8 weights during training, providing a minimum reproducible example (MRE) and additional information about your current setup would be extremely helpful. This ensures we can give you the most comprehensive support possible 🌟.

If this 🐛 issue relates to a bug, a detailed MRE would allow us to address it more efficiently. You can find guidance on creating an MRE [here](https://docs.ultralytics.com/help/minimum_reproducible_example/).

We strive to reply to all inquiries promptly—thanks for your patience and enthusiasm! 🔍👨‍💻

**Comment by pderrenger** [comment]
@wlangkau hello! 😊

Great question about using Ultralytics Cloud for transfer learning with your custom dataset. You can definitely leverage the Ultralytics Cloud to perform transfer learning without losing the pre-trained weights from models like `yolo11x.pt`.

Here's how you can do it:

1. **Upload Your Dataset**: First, ensure your dataset (images, labels, and `data.yaml`) is uploaded to the Ultralytics HUB. You can do this by navigating to the Datasets section in the HUB.

2. **Select Pre-trained Model**: When setting up your training in the Ultralytics HUB, you can choose a pre-trained model like `yolo11x.pt`. This will ensure that the model starts with the weights trained on the COCO dataset.

3. **Configure Training**: In the training configuration, you can specify your custom dataset by selecting it from the list of available datasets in the HUB. This will automatically reference your dataset in the cloud environment, so you don't need to manually adjust paths like you would in Colab.

4. **Start Training**: Once everything is set up, you can start the training process directly in the Ultralytics Cloud. This will utilize the cloud's hardware resources, saving you from additional costs on platforms like Google Colab.

The Ultralytics HUB interface is designed to make this process seamless, so you won't need to manually code the dataset paths as you do in Colab. Simply follow the steps in the HUB to configure and start your training session.

If you encounter any issues or have further questions, feel free to reach out. Happy training! 🚀
